examples/Mediawiki/SiteStatistics.yaml

```diff
@@ -0,0 +1,60 @@
+suite: Site Statisitics
+description: Testing site statistics' edits value
+type: RPC
+
+use-fixtures:
+  - Alice
+
+tests:
+  - description: Edit value should increase in site statistics after editing a page
+    default-session: Alice
+    retry:
+      tries: 3
```

Comment:
The retry value belongs into the individual step (request/response pair), not the test. Repeating the test isn't too useful, the point is that the effect of one request may take a while, so the subsequent one may fail at first and needs to be retried.